pages/sign_in_page.py

```diff
@@ -1,23 +1,23 @@
 from methods import Methods
 from locators import Locators
-from pages.home_page import HomePage
 
-class SignInPage(HomePage):
-    """Inherits click_icon method from HomePage"""
+
+class SignInPage():
+    """Sign in so far, Sign Up expected here as well"""
     def __init__(self, driver):
         self.driver = driver
         self.methods = Methods(self.driver)
         self.locators = Locators
 
-    def clear_box(self, box_name):
-        self.methods.clear_element('id', self.locators.ID_LOC[box_name])
+    def clear_boxes(self):
+        self.methods.clear_element(self.locators.EMAIL)
+        self.methods.clear_element(self.locators.PASSWORD)
 
-    def send_keys(self, box_name, keys):
-        self.methods.send_keys('id', self.locators.ID_LOC[box_name], keys)
+    def enter_credentials(self, entry):
```

Comment:
My advice is to use explicit params, username and password.

Reply:
???

Reply:
entry -> username and password as string values